workspaces/keycloak/package.json

```diff
@@ -18,6 +18,7 @@
     "lint": "backstage-cli repo lint --since origin/main",
     "lint:all": "backstage-cli repo lint",
     "prettier:check": "prettier --check .",
+    "prettier:fix": "prettier --ignore-unknown --write .",
```

Comment:
Is there a reason you need to do this? What is currently in place works for all the other plugins.